code/LINDA/LINDA_system.dm

```diff
@@ -64,7 +64,7 @@ turf/CanPass(atom/movable/mover, turf/target, height=1.5)
 	atmos_adjacent_turfs_amount = 0
 	for(var/direction in cardinal)
 		var/turf/T = get_step(src, direction)
-		if(!istype(T))
+		if(!issimulatedturf(T))
```

Comment:
Curious about this change, when LINDA can make air changes with non-simulated turfs (it simulates them).

Reply:
Does it? I thought the whole point of unsimulated turfs was that they don't process atmos (or maybe I misunderstood what they entail)? I figured it'd be fine, because most of the other atmos calculations like GetAtmosAdjacentTurfs only work with simulated turfs. 

This was a necessary change to prevent Lavaland from runtiming pretty hard, because it'd try to calculate atmos for turfs on the north-eastern edge of the map, next to the Necropolis wall. IIRC it tried to actually calculate atmos with either the walls or the outside of the map.

Reply:
https://github.com/ParadiseSS13/Paradise/blob/master/code/LINDA/LINDA_turf_tile.dm#L156

https://github.com/ParadiseSS13/Paradise/blob/master/code/LINDA/LINDA_turf_tile.dm#L200

It does a mimc call if it's not a simulated turf.

Reply:
Hm. Could you show me the area on the map that's giving you issues and mark which turfs are simulated or not?

Reply:
I did some testing, and it's resolved now. It probably got fixed when I moved the Necropolis turfs from /unsimulated/ to /simulated/indestructible. I'll revert that change.

Reply:
Ok, great, thanks. Yeah...uh, LINDA should handle that kind of thing gracefully, which is why I was interested in why it was runtiming.